crates/fluvio-extension-common/src/output/table.rs

```diff
@@ -13,6 +12,18 @@ pub trait TableOutputHandler {
     fn errors(&self) -> Vec<String>;
 }
 
+pub trait DisplayTable {
```

Comment:
This is also defined in crates/fluvio-extension-common/src/output/mod.rs,

You can just keep that there, and here you can import that trait and then you should be able to use `print_std` here